packages/tupaia-web-server/src/app/createApp.ts

```diff
@@ -49,11 +49,11 @@ export function createApp(db: TupaiaDatabase = new TupaiaDatabase()) {
       handleWith(routes.DashboardsRoute),
     )
     .post<routes.ExportDashboardRequest>(
-      'dashboards/:projectCode/:entityCode/:dashboardName/export',
+      'dashboards/:projectCode/:entityCode/:dashboardCode/export',
```

Comment:
Chose to switch to using `dashboardCode` here since it was consistent with the future plans for the subscribe/unsubscribe routes and generally feels cleaner than using name.